docs/Advanced-Concepts/Architectural-Checkpoints.rst

```diff
@@ -29,10 +29,34 @@ Loading Checkpoints in RTL Simulation
 --------------------------------------
 
 Checkpoints can be loaded in RTL simulations with the ``LOADARCH`` flag.
-The target config **MUST** use dmi-based bringup (as opposed to the default TSI-based bringup), and support fast ``LOADMEM``.
-The target config should also match the architectural configuration of however spike was configured when generating the checkpoint.
+The target config needs the following properties:
+
+- **MUST** use DMI-based bringup (as opposed to the default TSI-based bringup)
+- **MUST** support fast ``LOADMEM``
+- Should match the architectural configuration of however Spike was configured when generating the checkpoint (i.e. same ISA, no PMPs, etc).
```

Comment:
Does this PMP thing really cause problems if it differs between the spike used to generate the checkpoint and the DUT?

Reply:
For me, this fixed an issue where if an address is under `0x80200000` (where the payload is put in OpenSBI), then it would return an insn value of `0` causing an OpenSBI exception that wouldn't get resolved.